### DISCUSSION TITLE
[ASK] Minimum hardware requirement support Supabase for 1M users?

### DISCUSSION THREAD

**Discussion by nguyenthanhtien** [supabase/supabase#19556] [Questions]
I need develop web and mobile support 1M user. Supabase can support for 1M users. What's minimum hardware requirement support for 1M users?

**Comment by GaryAustin1** [comment]
I'm not sure you could get an answer with better info.  But to ask about supporting 1M users is sort of useless.   I would think things like how many users active at a time, what they are doing as far as database calls, storage calls, etc. would all come into play.   You don't even say if this is Supabase hosted or self-hosted.  

But at same time I don't know that providing more data will get an answer even then as there are so many variables.

This link shows some of the Supabase instance sizes just to see the range... https://supabase.com/docs/guides/platform/compute-add-ons
